apps/shopify/src/utils/base64.js

```diff
@@ -0,0 +1,25 @@
+export const convertBase64ToString = (str) => {
+  try {
+    const decodedId = window.atob(str);
+    return decodedId;
+  } catch (error) {
+    return null;
+  }
+};
+
+export const convertStringToBase64 = (str) => {
+  try {
+    const undecodedId = window.btoa(str);
+    return undecodedId;
+  } catch (error) {
+    return null;
+  }
+};
+
+export const convertToBase64 = (res) => {
```

Comment:
Still not a good name. Without typescript it is very unclear what this does. How about `convertProductToBase64` or `convertCollectionToBase64`.

Can we maybe make this two functions if we cover multiple use cases (product vs collection vs product variant).

Reply:
I was trying to make it generic as it was suggesting that we are converting the response to Base64 for whatever use cases(product vs collection vs product variant).
But yeah making separate functions will bring more clarity